docs/src/main/sphinx/connector/base-jdbc.rst

```diff
@@ -0,0 +1,194 @@
+===================
+JDBC connector base
+===================
+
+A number of connectors use JDBC connections to the data source. Typically
+specific JDBC drivers for the relevant data source are used, and numerous
+extensions in the connector ensure that the specifics of the data sources are
+taken into account. They include variations in data type mappings, support for
+specific SQL keywords and functions, and other aspects.
+
+The following connectors use the JDBC connector base:
+
+* :doc:`clickhouse`
+* :doc:`druid`
+* :doc:`mysql`
+* :doc:`oracle`
+* :doc:`postgresql`
+* :doc:`redshift`
+* :doc:`memsql`
+* :doc:`sqlserver`
```

Comment:
phoenix too

however, i don't believe this list will be kept up to date.
is it needed at all?

Reply:
Its not needed but might be good to have .. I am fine with dropping it .. wdyt @electrum ?

Reply:
Removed